packages/common/editors/EditorContainer.js

```diff
@@ -349,15 +335,22 @@ class EditorContainer extends React.Component {
   };
 
   render() {
-    const { width, height, column } = this.props;
-    const zIndex = isFrozen(column) ? zIndexes.FROZEN_EDITOR_CONTAINER  : zIndexes.EDITOR_CONTAINER;
-    const style = { position: 'fixed', height, width, zIndex, transform: this.calculateTransform() };
+    const { width, height, column, position } = this.props;
```

Comment:
This `position` under this context in `render()` might be confusing when the dev first look at the code, maybe we can rename it just in this render function? Because `position` is a valid attr in `style`, while we are assigning `...position` to the style as well. But I am OK if we don't change this.

Reply:
I have removed `position` and added `left/top` props instead